### ISSUE TITLE
[Question]: Can you clarify when updated_at field modified in "Change" event (in relation to changing from draft : false to draft:" 

### ISSUE THREAD

**Issue by leonid-deriv** [change-metrics/monocle#1115]
List of events - updated_at does not correspond to any other event so what triggered the update?

![image](https://github.com/change-metrics/monocle/assets/121090469/ddb62d16-2660-4c3a-9725-323a7e389f61)

**Comment by morucci** [comment]
`created_at` is the date when the event happened (for a `Change` it is the same value as `ChangeCreatedEvent`). `updated_at` only reported the last update date of a `Change`. When a `Change` is updated Monocle extracts all related Events and does not have the capability to reports when the Events have been updated.

**Comment by leonid-deriv** [comment]
Sorry, not sure I understand your last comment.
what are events which trigger updates? I can think of a couple (at least):
- change draft status - it will be great to have an event for this
- status changed (MERGED or CLOSED) - we have events already.

The reason we are interested in the draft field is because it can be used to identify when the actual review started.

**Comment by morucci** [comment]
Monocle only relies on the Change update date reported by the code review system. It does not make assumption about what kind of events are 'update' events.

Regarding Github, if we want to a new Event about draft, we need to check:

- if the Change's update date field is updated when the Change's draft status is changed.
- if the Github GraphQL schema have a notion change to draft/to ready event.

**Comment by leonid-deriv** [comment]
Yes, update date field is updated and there is also record in timelineItems. Here the request and response

```
query {
  repository(owner: "xxx", name: "xxxx) {
    pullRequest(number: 18) {
      id
      title
      isDraft
      createdAt
      updatedAt
      author {
        login
      }
      timelineItems(last: 5, itemTypes: [READY_FOR_REVIEW_EVENT]) {
        nodes {
          __typename
          ... on ReadyForReviewEvent {
            createdAt
            actor {
              login
            }
          }
        }
      }
    }
  }
}
```

```
{
  "data": {
    "repository": {
      "pullRequest": {
        "id": "PR_kwDOK5E-T85mxWkF",
        "title": "test for draft-pr",
        "isDraft": false,
        "createdAt": "2024-02-13T16:10:23Z",
        "updatedAt": "2024-02-13T16:19:31Z",
        "author": {
          "login": "xxx-xxx"
        },
        "timelineItems": {
          "nodes": [
            {
              "__typename": "ReadyForReviewEvent",
              "createdAt": "2024-02-13T16:19:31Z",
              "actor": {
                "login": "xxx-xxx"
              }
            }
          ]
        }
      }
    }
  }
}
```

**Comment by morucci** [comment]
Thank you for the investigation ! Then it might be possible to create that new Monocle Event for ReadyForReview. I can't tell when I'll be able to implement this but for sure that's an interesting data to have.

**Comment by leonid-deriv** [comment]
It will be great to have it.
